src/rules/noShadowedVariableRule.ts

```diff
@@ -63,8 +63,8 @@ export class Rule extends Lint.Rules.AbstractRule {
         `,
         optionsDescription: Lint.Utils.dedent`
             You can optionally pass an object to disable checking for certain kinds of declarations.
-            Possible keys are \`"class"\`, \`"enum"\`, \`"function"\`, \`"import"\`, \`"interface"\`, \`"namespace"\`, \`"typeAlias"\`
-            and \`"typeParameter"\`. Just set the value to \`false\` for the check you want to disable.
+            Possible keys are \`"class"\`, \`"enum"\`, \`"function"\`, \`"import"\`, \`"interface"\`, \`"namespace"\`, \`"typeAlias"\`,
+            \`"typeParameter"\` and \`"underscore"\`. Just set the value to \`false\` for the check you want to disable.
```

Comment:
The rest of the types in this list are node types, whereas `"underscore"` refers to the text of the variable's name.

```suggestion
            , and \`"typeParameter"\`. You can also pass \`"underscore\`" to ignore variable names that begin with `_`. Just set the value to \`false\` for the check you want to disable.
```

Reply:
_(I think that suggested change will end up with a duplicate `,` by the way...)_

Reply:
Good suggestion.